src/courseware/course/new-sidebar/sidebars/discussions-notifications/notifications/NotificationsWidget.jsx

```diff
@@ -68,22 +73,32 @@ const NotificationsWidget = () => {
 
   return (
     <div className="border border-light-400 rounded-sm" data-testid="notification-widget">
-      <UpgradeNotification
-        offer={offer}
-        verifiedMode={verifiedMode}
-        accessExpiration={accessExpiration}
-        contentTypeGatingEnabled={contentTypeGatingEnabled}
-        marketingUrl={marketingUrl}
-        upsellPageName="in_course"
-        userTimezone={userTimezone}
-        shouldDisplayBorder={false}
-        timeOffsetMillis={timeOffsetMillis}
-        courseId={courseId}
-        org={org}
-        upgradeNotificationCurrentState={upgradeNotificationCurrentState}
-        setupgradeNotificationCurrentState={setUpgradeNotificationCurrentState}
-        toggleSidebar={() => toggleSidebar(currentSidebar, WIDGETS.NOTIFICATIONS)}
-      />
+      <PluginSlot
+        id="notification_widget_plugin"
```

Comment:
We're trying to standardize how slots are exposed (details here: https://github.com/openedx/wg-frontend/issues/178).  I won't ask you to tackle all that here (we'll probably do it ourselves in [this other PR](https://github.com/openedx/frontend-app-learning/pull/1381)), but in order to avoid a breaking change to the slot ID, do you mind changing it as suggested below?

```suggestion
        id="notification_widget_slot"
```

Reply:
ahh sounds good I can rename both of these